### PR TITLE
Implemented product page tests.

### PR DESCRIPTION
**Pull request by andkochergin**

### CODE REVIEW

File: spec/features/product_page_spec.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  it 'shows product image gallery' do
+    expect(all('.product-gallery__thumb').any?).to eq(true)
+  end
+
+  it 'shows product main image' do
+    expect(all('.offers-description__image').any?).to eq(true)
+  end
+
+  it 'add to cart button works' do
```
Comment (yahorzhylinskilendeavor) [inline spec/features/product_page_spec.rb]: i think here you can use new describe.
Right now in console you will something like:
```
Onliner product page
  add to cart button works
```

If add new describe:

```ruby
describe 'add to cart button' do
  it 'adds a product to cart' do
    # test code....
  end
end
```

Than in console you will see:
```
Onliner product page
  add to cart button
    adds a product to cart
```

Comment (yahorzhylinskilendeavor) [reply spec/features/product_page_spec.rb]: For my view with new describer it looks better. What do you think?

Comment (andkochergin) [reply spec/features/product_page_spec.rb]: Yes, I agree with you

Comment (andkochergin) [reply spec/features/product_page_spec.rb]: done

**Pull request merged**